package ExceptionHandling;

public class Ex5 {
    public static void main(String[] args) {
        int[] arr={1,2,3,4,5,6};
        try{
            System.out.println();
        }catch(ArithmeticException e){
            System.out.println("None");
        }
    }
}
